config/7.10.0/common.yaml

```diff
@@ -33,6 +33,7 @@ console:
               app: "[[.ApplicationName]]"
               application: "[[.ApplicationName]]"
               service: "[[.ApplicationName]]-[[.Console.Name]]"
+              cluster: "kubernetes.ping.console-cluster"
```

Comment:
add it on the dashbuilder template as well.

Reply:
and, can't we use a smaller name?
or use kubernetes.ping.... is required by jgroups?

Reply:
We are free to use every name, k8s.ping.console sounds better ?

Reply:
console and all components should use the templated names as part of cluster name. this is because our operator supports the deployment of multiple kieapps in a single namespace. we need to be sure they aren't communicating when the intention is a separate install.

Reply:
@tchughesiv  jgrp.k8s.<console-kieserver-dashbuilser-etc>.[[.ApplicationName]] ?